app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorPrimary">#FF837D87</color>
```

Comment:
what the fuck is going on here

Reply:
I mean with the 32-bit color instead of regular 24-bit color

Reply:
huh i didn't notice that. i just copied the topaz color from ColorPickerFragment. what do i do to make it right

Reply:
probably drop the leading FF which leaves [#837D87](https://color-hex.org/color/837d87)